contracts/evoting/types/ballots_test.go

```diff
@@ -468,5 +469,72 @@ func TestSubject_IsValid(t *testing.T) {
 
 	valid = configuration.IsValid()
 	require.False(t, valid)
+}
 
+func TestBallot_Equal(t *testing.T) {
```

Comment:
I bet you know what I'm thinking 😅 but yeah, splitting into smaller test execution units would be an improvement.
In particular, this would be a perfect candidate for a table-driven test.